packages/app/src/systems/Sidebar/components/Menu/Menu.test.tsx

```diff
@@ -40,15 +40,15 @@ describe('Menu', () => {
     await testA11y(<Menu items={ITEMS} />, { wrapper: TestWrapper });
   });
 
-  it('should navigate to path when click', async () => {
+  it.skip('should navigate to path when click', async () => {
```

Comment:
Why do we need to skip this?

Reply:
I am ok skipping / removing those as they're already covered in e2e